src/request_handlers/shop.rs

```diff
@@ -51,7 +51,10 @@ pub enum ShopResponse {
         error: String,
     },
     Update {
+        /// List of new app ids.
         app_infos: Vec<AppInfo>,
+        /// List of removed app ids.
```

Comment:
please document more verbosely: removed versions or removed all-versions of an app_id?

Reply:
Sorry, this should be `Removed appinfos` or so. It is a list of ids (row-ids to be explicit) which are no longer active. (active = false)